### ISSUE TITLE
Meta: Dashboard

### ISSUE THREAD

**Issue by timoludwig** [Integreat/cms-django#133]
The dashboard is the landing page after the successful login.
There should be summary cards with information about different areas of the backend.
Here is a list of all card which should be there:

- #1 Analytics overview in dashboard
- #2 Show latest feedback in dashboard
- #3 Activated extras in Dashboard
- #4 Show translation coverage in dashboard
- #5 Integrate RSS reader in dashboard
- #6 Chat for authors
- #78 Dashboard: API download size traffic light

**Comment by svenseeberg** [comment]
We were thinking about a To-Do Dashboard and I think that would incorporate the HIX-value. 
Here are our ideas for this dashboard contentwise, though nothing completely set in stone yet, just wanted to share so we are all on the same page: 

Idea: Maximum 5 items (maybe 7?) per day that are displayed, but there is a "load more tasks" button.

The following activities could be displayed and completed by the municipalities:

- unread feedback
- Faulty links
- Outdated pages (from 365 days, but can also be done earlier)
- Pages awaiting revision (always display first)
- Too large images or pages with too much content
- Pictures without alternative texts --> maintain texts
- POIs without categories
- Pages with HIX value below 10
- Inactive users who have not logged in for 1 year
- Missing icons on the non-bottom level

optionally expandable at some point:

- Possible duplicates for POIs
- Outdated translations that could also be made DeepL if budget is available

What should not go in:

- Live content that has changed at some point

**Comment by svenseeberg** [comment]
@osmers can the service & UX/UI teams provide an example/design?

**Comment by osmers** [comment]
@svenseeberg yes, our idea was to have a design ready by mid April but depending on how fast the UI/UX team is, maybe earlier :)

**Comment by ulliholtgrave** [comment]
As this would mean quite a big redesign of our dashboard, I would suggest to also evaluate the current state of the dashboard. 
Maybe we could move the chat and RSS feed to another menu item "Integreat/Malte Community" and we could just add a very small widget to the main dashboard indicating that something new (New RSS entry/new chat message) showed up.

**Comment by osmers** [comment]
@ulliholtgrave I think that would then be part of the design process, right?

**Comment by svenseeberg** [comment]
As it has not been mentioned yet here:

The goal of the dashboard should be to incite content editors to do something.

That means the Dashboard should provide actionable information.

**Comment by osmers** [comment]
Took #78 out of the list and added the label topic:dashboard instead. This label will be added to all further issues concerning the dashboard
